stages/org.osbuild.livesys

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/python3
+"""
+Configure basic aspects of the `/etc/sysconfig/livesys` file, necessary for
+live installations.
+"""
+
+import os
+import sys
+
+import osbuild.api
+
+SCHEMA_2 = """
+"options": {
+  "additionalProperties": true,
+  "required": ["livesys_session"],
+  "properties": {
+    "livesys_session": {
+      "type": "string",
+      "description": "Livesys session to use",
+      "items": {
+        "type": "string"
+      }
+    }
+  }
+}
+"""
+
+PATH = "/etc/sysconfig/livesys"
+
+CONFIG = """
+livesys_session="{livesys_session}"
+"""
+
+
+def main(tree, options):
+    livesys_session = options["livesys_session"]
+
+    with open(PATH, "w", encoding="utf8") as f:
+        f.write(CONFIG.format(livesys_session=livesys_session))
+
+    os.fchown(f.fileno(), 0, 0)
```

Comment:
Is `chown` and `chmod` really needed? IIRC, `root:root` and `644` is the default for files, and we just depend on this fact in other stages.

I might be wrong, though, feel free to correct me. :)